Automattic/vip-go-mu-plugins
# Replace "host" field by "http_host" in log2logstash messages

## Description

On Kubernetes we have a common fluent-bit and log-courier configuration for all pods, and log2logstash JSON messages are confusing it because of the `host` field. In this PR we are changing it for `http_host`, which defines better its content and avoids the problem with logging configuration.

## Changelog Description

### Logging changes in log2logstash

We have changed a field in the log2logstash message to avoid the Kubernetes logging system getting confused by it.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Send a testing log2logstash message with: `wp eval '\Automattic\VIP\Logstash\log2logstash( array( "severity" => "info", "feature" => "log_testing", "message" => "This is a testing message", "extra" => "Extra payload" ) );'`
1. Ensure that the generated log contains the field `http_host` instead of `host`
